references/cli/status-ui/model/info.go

```diff
@@ -0,0 +1,67 @@
+package model
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/oam-dev/kubevela/apis/types"
+	"github.com/oam-dev/kubevela/pkg/utils/helm"
+	"github.com/oam-dev/kubevela/version"
+)
+
+type Info struct {
+	cluster string
+}
+
+const (
+	UNKOWN_VERSION = "UNKNOWN"
+)
+
+func NewInfo() *Info {
+	return &Info{
+		cluster: "local",
```

Comment:
There is one const named "ClusterLocalName" in multicluster.go。 It is better to use it , not String "local"

Reply:
ok, I have fixed it.